src/hardware/vga_xga.cpp

```diff
@@ -882,10 +902,25 @@ void XGA_BlitRect(Bitu val) {
 					break;
 			}
 
-			const Bitu destval = GetMixResult(mixmode, srcval, dstdata);
-			//LOG_MSG("XGA: DrawPattern: Mixmode: %x Mixselect: %x", mixmode, mixselect);
+			bool doit = true;
```

Comment:
Can we find a better name? 😆

Reply:
Check the next commit that refactor it (I also wasn't a fan of `doit` :laughing: )

Reply:
Ah yeah, I reviewed it commit by commit and somehow missed the refactoring step.